src/main/proto/Report.proto

```diff
@@ -60,4 +65,24 @@ message Schedule {
 message ReportLog {
     string task_status = 1;
     string task_executed = 2;
+    string threshold_met = 3;
```

Comment:
I am not sure it's a good idea to have different coding style for messages. Some fields are camelCase, and some a re snake_case. I think we should use only one or another.

Reply:
Mostly we have snake_case in property name
use  camelCase when define custome message